spec/models/user_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject do
+    User.new(name: 'Anything', bio: 'Lorem ipsum', photo: 'me.jpg',
+             PostsCounter: 1)
+  end
+  before { subject.save }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
 end
```

Comment:
- [ ] Kindly make sure the Your User modal contain validation for example :point_down: 

```rb
class User < ApplicationRecord

  validates :name, presence: { message: 'Name must not be blank' }
  validates_numericality_of :posts_counter, only_integer: true, greater_than_or_equal: 0

  def top_most_recent_posts(limit = 3)
    posts.order(created_at: :desc).limit(limit)
  end
end
```

- [ ] It would be better also to render validation errors in the correspondent view.